### PR TITLE
Display compiler generated members in raw view.

### PR DESCRIPTION
**Pull request by tmat**
The debugger has an option 
image

which is also available via `raw` modifier (`<expr>, raw`). 

The meaning for managed objects right now is to disable using debugger visualizer attributes on the object instance. For example, an instance of `Dictionary<string,string>` is displayed like so:

image

This change shows all compiler-generated members in this view.

An iterator looks like so:
image

An async state machine like so:
image

Fixes https://github.com/dotnet/roslyn/issues/22428.

### CODE REVIEW

**Comment by tmat** [comment]
@cston @ivanbasov @r-ramesh PTAL

File: src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/MemberExpansion.cs [hunk 1]
```diff
@@ -58,25 +58,21 @@ internal sealed class MemberExpansion : Expansion
 
             var expansions = ArrayBuilder<Expansion>.GetInstance();
 
+            // Expand members. TODO: Ideally, this would be done lazily.
```
Comment (ivanbasov) [inline src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/MemberExpansion.cs]: GitHub issue for the TODO?

Comment (tmat) [reply src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/MemberExpansion.cs]: I just moved the comment up :)

**Comment by tmat** [comment]
@jinujoseph For approval.

File: src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/MemberExpansion.cs (hunk 1 above)
Comment (tmat) [reply src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/MemberExpansion.cs]: (will file an issue)

**Comment by sharwell** [comment]
@tmat attention to detail in debugging like this was always one of the reasons why I love the VS debugger. Thanks for this!

**Pull request merged**